src/components/atoms/InputField/InputField.tsx

```diff
@@ -58,6 +58,11 @@ interface InputFieldProps
   name?: string;
 }
 
+/**
+ * @deprecated Please use the appropriate attendee or admin Input
+ * @see src/components/admin/Input
+ * @see src/components/attendee/Input
+ */
```

Comment:
We should work to retire this component in favor of one of the two flavors provided in the comment